src/sparseml/modifiers/utils/gpt_helpers.py

```diff
@@ -34,7 +37,40 @@
 torch.backends.cudnn.allow_tf32 = False
 
 
-class SparseGPT:
+class GPT(ABC):
```

Comment:
I'm not sure "GPT" makes much sense as a name here. Maybe "TerminalModuleCompressor" or something?